next/frontend/utils/formState.ts

```diff
@@ -1,4 +1,5 @@
 import { GenericObjectType, retrieveSchema, RJSFSchema } from '@rjsf/utils'
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
I ran into this problem with the current eslint project setup. I didn't want to install unnecessary packages or add a rule to eslint, so I threw it into ignor. What to do with it?

Reply:
The library not being in package is an issue, feel free to add it. You can do it in a separate PR, bude I don't mind doing it in here as well.